src/dom/emittermixin.js

```diff
@@ -93,11 +91,15 @@ const DomEmitterMixin = extend( {}, EmitterMixin, {
 				return;
 			}
 
-			args[ 0 ] = proxy;
+			emitter = proxy;
 		}
 
 		// Execute parent class method with Emitter (or ProxyEmitter) instance.
-		EmitterMixin.stopListening.apply( this, args );
+		EmitterMixin.stopListening.call( this, emitter, event, callback );
+
+		if ( emitter instanceof ProxyEmitter ) {
+			emitter.detach( event );
```

Comment:
AFAICS, `event` can be undefined. Which means calling `emitter.detach()` but its first param is not optional. Could you verify that it all works fine in cases where `stopListening()`'s params are not passed?

Reply:
You mean besides tests? The `event` parameter is check by the `EmitterMixin.stopListening()` and also in here `dom/EmitterMixinProxy.detach()`: https://github.com/ckeditor/ckeditor5-utils/pull/190/files/86c8efc9cff0415a01cdd3cd2a29aef81e1aadce#diff-99d3cd5d9460b6839252663e7c38b1c6R218 - actually it is checked if given `event` is defined in `_domListeners` so if it is falsy then `detach` will do nothing.

Reply:
It's handled very poorly there, because if event is `undefined` then we do `listeners[ undefined ]`. And the API docs of `detach()` are also incorrect.

Reply:
In other words – if we expect `event` to be undefined then it should be handled explicitly. However, it's not the problem of your PR – it was like that in the past.

Reply:
Actually, it seems to be a bug: https://github.com/ckeditor/ckeditor5-utils/issues/199